rqt_py_common/src/rqt_py_common/layout_util.py

```diff
@@ -34,11 +34,12 @@
 
 from python_qt_binding.QtCore import Qt
 from python_qt_binding.QtWidgets import QWidgetItem, QSpacerItem
-import roslib
-import rospy
+
+from rclpy import logging
 
 
 class LayoutUtil(object):
+    _logger = logging.get_logger("LayoutUtil")
```

Comment:
Should use single quotes.

Same below.

Reply:
Seems subjective. My understanding is that single and double quotes are interchangeable in python